workers/data_refinery_workers/processors/management/commands/qn_dispatcher.py

```diff
@@ -0,0 +1,86 @@
+"""This command will create and run survey jobs for each experiment
+in the experiment_list. experiment list should be a file containing
+one experiment accession code per line.
+"""
+
+import boto3
+import botocore
+import nomad
+import uuid
+
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+from nomad.api.exceptions import URLNotFoundNomadException
+
+from data_refinery_common.logging import get_and_configure_logger
+from data_refinery_common.job_lookup import ProcessorPipeline, SurveyJobTypes
+from data_refinery_common.message_queue import send_job
+from data_refinery_common.models import SurveyJob, SurveyJobKeyValue
+from data_refinery_common.utils import parse_s3_url, get_env_variable
+
+from data_refinery_common.models import (
+    ComputationalResult,
+    ComputedFile,
+    Dataset,
+    Experiment,
+    ExperimentOrganismAssociation,
+    ExperimentSampleAssociation,
+    ExperimentSampleAssociation,
+    Organism,
+    OrganismIndex,
+    ProcessorJob,
+    ProcessorJobDatasetAssociation,
+    Sample,
+    SampleComputedFileAssociation,
+)
+
+logger = get_and_configure_logger(__name__)
+
+MIN = 100
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        """ Handle it! """
+
+        organisms = Organism.objects.all()
+
+        for organism in organisms:
+            samples = Sample.processed_objects.filter(organism=organism, has_raw=True, technology="MICROARRAY", is_processed=True)
+            if samples.count() < MIN:
+                logger.error("Proccessed samples don't meet minimum threshhold",
+                    organism=organism,
+                    count=samples.count(),
+                    min=MIN
+                )
+                continue
+
+            platform_counts = samples.values('platform_accession_code').annotate(dcount=Count('platform_accession_code')).order_by('-dcount')
+            biggest_platform = platform_counts[0]['platform_accession_code']
```

Comment:
This biggest_platform may have less than 100 samples. We should decide whether or not to make a qn target based on each organism's largest platform, not their full sample count.